Screenbox/Pages/Search/SearchResultPage.xaml

```diff
@@ -34,6 +34,12 @@
                     Icon="{ui:FontIcon FontFamily={StaticResource ScreenboxSymbolThemeFontFamily},
                                        Glyph={StaticResource PlayAddGlyph}}"
                     Text="{strings:Resources Key=PlayNext}" />
+                <MenuFlyoutItem
+                    Command="{x:Bind Common.AddToQueueCommand}"
+                    CommandParameter="{Binding}"
+                    Icon="{ui:FontIcon FontFamily={StaticResource ScreenboxSymbolThemeFontFamily},
+                                       Glyph={StaticResource PlayAddGlyph}}"
+                    Text="{strings:Resources Key=AddToQueue}" />
```

Comment:
I would do something like this:

```suggestion
                <MenuFlyoutSubItem Icon="Add" Text="Add to">
                    <MenuFlyoutItem
                        Command="{x:Bind Common.AddToQueueCommand}"
                        CommandParameter="{Binding}"
                        Icon="{ui:FontIcon FontFamily={StaticResource ScreenboxSymbolThemeFontFamily},
                                           Glyph={StaticResource PlaylistMusicGlyph}}"
                        Text="{x:Bind strings:Resources.PlayQueue}" />
                </MenuFlyoutSubItem>
```

Designed to be expandable, allowing us to add new features (such as adding to a playlist) in the future.

Reply:
I recognize this pattern is from the inbox Media Player, but I have to disagree with this. I think "Add to play queue" should be a top-level option, same as "Play next". "Add to playlist" will also be a top-level option with a sub-menu to a list of playlists or an option to create a new playlist.